inventory/api/compute/v1/compute.proto

```diff
@@ -645,6 +645,7 @@ message InstanceResource {
   repeated WorkloadMember workload_members = 30 [(ent.edge) = {ref: "instance"}]; // back-reference to the Workload Members associated to this Instance
   provider.v1.ProviderResource provider = 40 [(ent.edge) = {unique: true}]; // Provider this Instance is provisioned through
   localaccount.v1.LocalAccountResource localaccount = 41 [(ent.edge) = {unique: true}]; // Local Account associated with this Instance
+  repeated CustomConfigResource custom_config = 42 [(ent.edge) = {ref: "instances"}]; // List of Custom Config associated with this Instance
```

Comment:
By convention the `ref` is used for back-references for other edges in Inventory. Can you reverse the edge? (put `ref` in the CustomConfig resource).
It should not change much in terms of autogenerated code or anything else.